.gitignore

```diff
@@ -16,6 +16,7 @@ _tools
 cpp/
 !cpp/CMakeLists.txt
 proto-cpp/
```

Comment:
```suggestion
```

Reply:
We still provide the script `scripts/generate-cpp.sh` for those project don't use CMake. I think we can keep this.

Reply:
Well.